lib/fetch-requests/foods-fetch-requests.js

```diff
@@ -0,0 +1,25 @@
+const url = `https://protected-basin-11627.herokuapp.com/`
+const handleResponse = require('../response-handlers/initial-response-handler')
+
+const getAllFoods = () => {
+  fetch(`${url}/api/v1/foods`)
+    .then(response => handleResponse(response))
+    .then(foods => renderFoods(foods))
+    .catch(error => console.error({ error }))
+}
+
+const renderFoods = (foods) =>  {
```

Comment:
Should this function live somewhere else?

Reply:
probably in the response handlers, but don't worry about refactoring it out so early on.  I think get the call working, then refactor.  Red to green, then break things out.